src/components/menus/FileExport.ts

```
@@ -0,0 +1,490 @@
+                case 'em':
+                case 'i':
+                    return `*${this.innerTextToMarkdown(element)}*`;
+                case 'u':
+                    return `<u>${this.innerTextToMarkdown(element)}</u>`;
+                case 'code':
+                    return element.parentElement?.tagName.toLowerCase() === 'pre' 
+                        ? this.innerTextToMarkdown(element)
+                        : `\`${this.innerTextToMarkdown(element)}\``;
+                case 'pre':
+                    // Determine language if possible
+                    const langMatch = element.className.match(/language-([^\s]+)/);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)